src/pop/mempool.cpp

```diff
@@ -36,6 +82,7 @@ PopData generatePopDataImpl(
 
     // try to fit ATVs
     auto& atvcandidates = block.second->atvs;
```

Comment:
hmm. I wonder if we can maintain max heap or something like that in `block.second->atvs`. Could you please explore?

Reply:
Could you elaborate? Do you want to collect ATVs from different VBK blocks?

Reply:
`block.second` contains a vector of ATVs. I wonder if we could maintain `atvs` as sorted array instead of sorting it every time.

Reply:
Interesting idea. I will take a look what data structure fits best for our case.

Reply:
I am pretty sure that "max heap" will work 
https://en.wikipedia.org/wiki/Min-max_heap